### DISCUSSION TITLE
Is it possible to use custom fonts on PaperMod?

### DISCUSSION THREAD

**Discussion by davidjnevin** [adityatelange/hugo-PaperMod#383] [Q&A] [answered]
I wish to include local fonts. Is this possible?

I have included the following  `<link rel="stylesheet" href="/fonts/font-family.css">` in `baseof.html`.
My fonts are located in the static/fonts folder on the root. Is this correct?
Do I need to include them somewhere else to see them when I run Hogo server? 
Thanks for your help.
David

edits to include code.

**Comment by davidjnevin** [comment, marked as answer]
## Okay, I fixed it.

Always the way, three days of hair pulling, reach out for help and then realize the problem.

### For anyone else having the same problem.

1. My fonts are stored locally in static/fonts/
1. My style sheet `font-family.css` with `@font-face`definitions etc are in that folder along with all my font files. The URL to the style sheet in Baseof.html is `<link rel="stylesheet" href="/fonts/font-family.css">`
2. My problem was I was inheriting fonts in another CSS.

**Reply by kdkasad** [reply]
While that works, I recommend using `extend_head.html` rather than `baseof.html`. It's designed to add custom HTML to the `head` of each page. That way you won't have to worry about updating your `baseof.html` and possibly having conflicts.

**Reply by tiancaigk** [reply]
Would you please post the code? I had the same problem. Thank you
